tests/main.test.ts

```diff
@@ -634,6 +632,7 @@ export function createContext(
     },
     octokit: new octokit.Octokit(),
     eventName: "issue_comment.created" as SupportedEventsU,
+    organizations: ["ubiquity"],
```

Comment:
Seems suspiciously wrong. Should be dynamic based on installations.

Reply:
it's a unit test